frontend/discourse/admin/templates/admin-config/login/discourse-id.gjs

```diff
@@ -0,0 +1,204 @@
+              </div>
+            </div>
+          </div>
+        </section>
+
+        {{! Statistics Card }}
+        <section class="admin-config-area-card">
+          <h3 class="admin-config-area-card__title">
+            {{icon "chart-bar"}}
+            {{i18n "admin.config.discourse_id.stats.title"}}
+          </h3>
+          <div class="admin-config-area-card__content">
+            <div class="discourse-id-stats">
+              <div class="discourse-id-stats__item">
+                <span class="discourse-id-stats__value">
+                  {{@model.stats.total_users}}
+                </span>
+                <span class="discourse-id-stats__label">
+                  {{i18n "admin.config.discourse_id.stats.total_users"}}
+                </span>
+              </div>
+              <div class="discourse-id-stats__item">
+                <span class="discourse-id-stats__value">
+                  {{@model.stats.signups_30_days}}
+                </span>
+                <span class="discourse-id-stats__label">
+                  {{i18n "admin.config.discourse_id.stats.signups_30_days"}}
+                </span>
+              </div>
+              <div class="discourse-id-stats__item">
+                <span class="discourse-id-stats__value">
+                  {{@model.stats.logins_30_days}}
+                </span>
+                <span class="discourse-id-stats__label">
+                  {{i18n "admin.config.discourse_id.stats.logins_30_days"}}
+                </span>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        {{! Advanced Accordion }}
+        <details class="admin-config-area-card discourse-id-advanced-accordion">
+          <summary class="admin-config-area-card__title">
+            {{i18n "admin.config.discourse_id.advanced.title"}}
+          </summary>
+          <div class="admin-config-area-card__content">
+            <div class="discourse-id-advanced">
+              {{! Regenerate Credentials }}
+              <div
+                class="discourse-id-advanced__row discourse-id-advanced__row--danger"
+              >
+                <div class="discourse-id-advanced__setting">
+                  <span class="discourse-id-advanced__label">
+                    {{i18n
+                      "admin.config.discourse_id.advanced.regenerate_credentials"
+                    }}
+                  </span>
+                  <span class="discourse-id-advanced__description">
+                    {{i18n
+                      "admin.config.discourse_id.advanced.regenerate_warning"
+                    }}
+                  </span>
+                </div>
+                <DButton
+                  @action={{this.regenerateCredentials}}
+                  @label="admin.config.discourse_id.advanced.regenerate_button"
+                  @disabled={{this.regenerateDisabled}}
+                  @isLoading={{this.isRegenerating}}
+                  class="btn-danger"
```

Comment:
I think we can use a `btn-default` here. This action isn't dangerous, it's not deleting any data. Unless there is an error, it shouldn't cause any problems or outage, right? (In fact, if there is an error during regeneration, it should exit and do nothing.)